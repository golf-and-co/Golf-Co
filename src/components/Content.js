import React from 'react'
import { StaticQuery, graphql } from 'gatsby'
import styled from 'styled-components'
import PropTypes from 'prop-types'

const Wrap = styled.section`
  background-color: #E4ECD9;
  padding:30px;

  p {
    max-width: 920px;
    margin: 20px auto;
    color: #000;
    text-align: center;

    @media (max-width: 768px) {
      margin: 5px auto;
    }
  }
`

const Content = ({ data }) => (
  <Wrap>
    <p>{data.description}</p>
  </Wrap>
)

export default Content

Content.propTypes = {
  data: PropTypes.object.isRequired,
}
