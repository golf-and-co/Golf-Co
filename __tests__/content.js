import React from "react"
import renderer from "react-test-renderer"

import Content from "../src/components/Content"

describe("Content", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Content data={{description:"test"}} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})