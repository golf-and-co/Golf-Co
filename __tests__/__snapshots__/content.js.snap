// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders correctly 1`] = `
<section
  className="sc-bdVaJa dHvjQq"
>
  <p>
    test
  </p>
</section>
`;
